packages/components/src/internal/components/base/models/Container.ts

```diff
@@ -34,4 +34,8 @@ export class Container extends Record(defaultContainer) implements Partial<ICont
     declare sortOrder: number;
     declare title: string;
     declare type: string;
+
+    hasActiveModule(moduleName: string): boolean {
```

Comment:
Nick and I discussed this briefly last week. I think we'll leave it for now so that this PR doesn't include any functional changes. Comment added to method.